website/pages/index.js

```diff
@@ -0,0 +1,209 @@
+import Head from 'next/head'
+
+export default function Home() {
+  return (
+    <div className="container">
```

Comment:
I added tailwind to the project. It will be great if we can use that. Or have a look at [tail-ui](https://github.com/udaypydi/tail-ui). A react component library based on tailwind css

Reply:
I don't have prior experience with TailwindCSS. I'll try to do what I can with this but you might have to take lead on this one.

Reply:
Sure. I will tae care of the styling and tail-ui components.